### ISSUE TITLE
Sub-list in 4.7.1 should be unordered

### ISSUE THREAD

**Issue by clapierre** [w3c/publ-a11y#591]
The sub-list in
4.7.3 Instructions

should be unordered.
ie.

  * IF flashing_hazard:
          THEN display "Flashing content".
  * IF motion_simulation_hazard:
          THEN display "Motion simulation".
  * IF sound_hazard:
          THEN display "Loud sounds".

**Comment by gregoriopellegrino** [comment]
Hi, I'm not clear on the request, in all cases where we have instructions, they are always inside `<ol>` tags...

**Comment by mattgarrish** [comment]
Closing as this was fixed with #667
